healme.lic

```diff
@@ -69,32 +73,44 @@ class HealMe
 
   def heal_wound(spells, part, internal)
     if @using_waggle
-      spells['Heal Wounds']['cast'] = "cast #{part}"
-      spells['Heal Wounds']['cast'] = "cast #{part} internal" if internal
-      check_discern(spells['Heal Wounds'], @settings) if spells['Heal Wounds']['use_auto_mana']
+      spells['Heal Wounds']['cast'] = "cast #{part.first['part']}"
+      spells['Heal Wounds']['cast'] = "cast #{part.first['part']} internal" if internal
+      data = check_discern(spells['Heal Wounds'], @settings) if spells['Heal Wounds']['use_auto_mana']
+      if data
+        maxmana = part.sum{ |x| if x['type'] =~ /Fresh/i;x['level'];else;0;end;} * 2
```

Comment:
❌ Please convert this to a multi-line block for readability (and consider `max_mana` as the var name).